Dockerfile

```diff
@@ -6,7 +6,7 @@ COPY . /src
 
 RUN hugo --environment ${TRAINING_HUGO_ENV} --minify
 
-FROM ubuntu:focal AS wkhtmltopdf
+FROM ubuntu:jammy AS wkhtmltopdf
 RUN apt-get update \
     && apt-get install -y curl \
     && curl -L https://github.com/wkhtmltopdf/packaging/releases/download/0.12.6-1/wkhtmltox_0.12.6-1.focal_amd64.deb --output wkhtmltox_0.12.6-1.focal_amd64.deb \
```

Comment:
Actions are failing for this PR. For it to work, I guess we also need to change the wkhtmltox to the jammy version. But I have not tested this yet.

Reply:
Build is now working :+1: